### ISSUE TITLE
Definitions for IC_TX_BUFFER_DEPTH inconsistent

### ISSUE THREAD

**Issue by fivdi** [raspberrypi/pico-sdk#335]
In the current develop branch, IC_TX_BUFFER_DEPTH is defined as 32 [here](https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2_common/hardware_i2c/include/hardware/i2c.h#L260):
```c
    const size_t IC_TX_BUFFER_DEPTH = 32;
```
and as 16 [here](https://github.com/raspberrypi/pico-sdk/blob/develop/src/rp2040/hardware_structs/include/hardware/structs/i2c.h#L132):
```c
// IC_TX_BUFFER_DEPTH ................ 16
```
I think 16 is the correct value.

**Comment by lurch** [comment]
Possibly related to #324 ? ping @Wren6991

**Comment by fivdi** [comment]
> Possibly related to #324 ?

Yes, most likely.

**Comment by fivdi** [comment]
Fixed by #381, closing.
